applications/DAOsign.md

```diff
@@ -0,0 +1,185 @@
+# DAOsign
+
+- **Team Name:** DAOsign 
+- **Payment Address:** 13s3c37rwrZfa8WQcZPdqE1zi1hGAguf2TWfWMr8f5aLBrg
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+
+### Overview
+
+DAOsign is a decentralized platform for agreements and policies with publicly verifiable and authorized cryptographic proofs and Governance Automation, designed for DAOs. 
+Traditional methods of signing agreements, such as paper-based systems or legacy e-signatures, are prone to fraud and lack transparency, and offer no solution for verification of the signer's identity or signer’s authority to execute the agreement. Instead these systems rely on costly legal proceedings for enforce agreements. 
+
+DAOsign provides publicly verifiable cryptographic proofs; secure, transparent and automated processing of complex Agreements and Policies. 
+
+Furthermore, DAOsign provides robust Governance and Policy Builder platform to provide governance, workflow automation and controls for organization policies and procedures.
+### Project Details
+A project goal is to store Agreements Proofs on Polkadot chain using Ink! smart contracts. 
+
+
+Deliveries include DAOsign platform with an ability to:
+- create agreements including generating using ChatGPT
+- add and remove signers and observers of different levels of anonymity
+- allow signing the documents with further storing of signature proofs on Polkadot blockchain.
+
+
+**Core DAOsign components:**
+
+<img width="902" alt="Screenshot 2023-05-27 at 16 51 43" src="https://github.com/Burmenska/Grants-Program/assets/3983591/e6e3e78d-edf4-4d50-a298-e01210d75963">
+
+
+**Technical design:**
+
+Proofs are implemented in the EIP-712 standard and adjusted to !ink ecosystem. Signed using user’s !ink private key. We are supporting this standard in Ink! smart contract to be able to verify the signature. Also, we are implementing an adjusted EIP-2771 standard to allow Agreement Signers gas-free delegation to publish proof on-chain.712 standard will be used for for proofs, adaptation means that user data is converted to a binary bit; we use its hash (serialization vs deserialization)
```

Comment:
I'm not sure what is meant by "signed using user's ink! private key". Wouldn't the user be signing it with their wallet? Which would mean that it is signed with the SS58 account private key.

Reply:
It was a typo, user signs proofs using their EVM private key. Also, users use their EVM wallet to sign in into website (dapp)